blockchain/consensus/deleg_cns/README.md

```diff
@@ -0,0 +1,225 @@
+# Delegated Consensus
```

Comment:
I love this README, amazing job!! :pray:

Reply:
Great! I added a version of the script that you can use as well, without Docker.